docs/ref/models/expressions.txt

```diff
@@ -425,6 +425,33 @@ Conditional expressions allow you to use :keyword:`if` ... :keyword:`elif` ...
 :keyword:`else` logic in queries. Django natively supports SQL ``CASE``
 expressions. For more details see :doc:`conditional-expressions`.
 
+SubQuery expressions
+--------------------
+
+.. currentmodule:: django.db.models.expressions
+.. class:: SubQuery(related_field, queryset, output_field=None)
+
+When a value is required that is dependent upon values from within a related
+object, it's possible to use an explicit ``SubQuery``. For example::
+
+    >>> from django.db.models.expressions import SubQuery
+    >>> queryset.annotate(largest_size=SubQuery('relation', Size.objects.order_by('-size')[:1]))
```

Comment:
This results in an AttributeError in `SubQuery.__init__ ` str object has no attribute all